test_performance.py

```diff
@@ -37,9 +77,15 @@ def printProgressBar (iteration, total, decimals = 1, fill = '█', printEnd = "
         total_style_predictions = {-1: 0, 0: 0, 1: 0}
         te_predictions = {-1: 0, 0: 0, 1: 0}
         total_predictions = {-1: 0, 0: 0, 1: 0}
+        sota_predictions = {-1: 0, 0: 0, 1: 0}
         for article in articles:
             total_count += 1
-            printProgressBar(total_count, 1173)
+            printProgressBar(total_count, 1373)
```

Comment:
what's that 1373?

Reply:
it is a magic number of all texts, that in our test data successfully pass all models. As some of our 1400 texts throw errors for the old tem